.circleci/config.yml

```
@@ -114,7 +114,9 @@ workflows:
     version: 2
     build:
         jobs:
-            context: dash-docker-hub
             - "python-3.6"
```

YAML can be a bit fiddly, but the colons are required (this happened to me as well).

```suggestion
            - "python-3.6":
```